### PR TITLE
Community page started, styling included.

### PR DESCRIPTION
**Pull request by bqanderson**
Good morning Yingrong,

I have push new changes to the migrate-gatsby branch, including new pages for /community and /clog (skeletons still at this point, but a starting place).

I also, made the changes you recommended in terms of the form CSS being in the _global.scss.

Please let me know your thoughts.

Thanks again!
BQA

### CODE REVIEW

File: src/components/blog/blog-navbar.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import React from 'react'
```
Comment (VinozzZ) [inline src/components/blog/blog-navbar.js]: Why does blog have a different nav-bar?

File: src/layouts/blog.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React, { Fragment } from 'react'
+
+import Banner from '../components/banner'
+import BlogNav from '../components/blog/blog-navbar'
+import Footer from '../components/footer'
+
+
+export default ({ children }) => (
+  <Fragment>
+    {/* <Banner /> */}
```
Comment (VinozzZ) [inline src/layouts/blog.js]: are we not using it?

File: src/styles/layouts/comm-layout.scss [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  @include media-breakpoint-up(lg) {
+    text-align: left;
+  }
+}
+
+.btn-forum {
+  padding: 0 $base-height * 7;
+  background-color: $aurora-blue;
+  color: $white;
+  font-family: 'Poppins';
```
Comment (VinozzZ) [inline src/styles/layouts/comm-layout.scss:40]: Should we also have SCSS variables for fonts?

File: src/styles/partials/_global.scss [hunk 4]
```diff
@@ -40,3 +40,22 @@ a {
...
+	display: block;
+	background: #fbfbfb;
+	border: 1px solid #eee;
+	border-radius: 4px;
+	color: #2683FF;
+	padding: 12px 18px 10px;
+	margin: 25px auto;
+	width: 100%;
+	max-width: 320px;
+  font-size: 16px;
```
Comment (VinozzZ) [inline src/styles/partials/_global.scss:60]: the indentation here looks off.

File: src/components/blog/blog-navbar.js (hunk 1 above)
Comment (bqanderson) [reply src/components/blog/blog-navbar.js]: The current site is using two different background approaches to the 'header' of each page. One is a light background (read white) with the blue logo and dark link text. The other is a dark background (blue or dark blue imagery) with white text and logo. I'll take a look to find a more semantic solution for the need of two navbar components.

File: src/layouts/blog.js (hunk 2 above)
Comment (bqanderson) [reply src/layouts/blog.js]: The banner is not currently on the blog site, but I wasn't sure if we wanted to keep it that way moving forward. Should I double check with DJ for how we want to handle this moving forward?

File: src/styles/layouts/comm-layout.scss (hunk 3 above)
Comment (bqanderson) [reply src/styles/layouts/comm-layout.scss:40]: Yeah, that's a damn good idea considering how often they change from one element to the next. Thanks for the suggestion!

File: src/styles/partials/_global.scss (hunk 4 above)
Comment (bqanderson) [reply src/styles/partials/_global.scss:60]: Hmmm... Indentation looks fine on my local file. I'll make sure to keep an eye out for this on the next commit. Good catch!

File: src/layouts/blog.js (hunk 2 above)
Comment (VinozzZ) [reply src/layouts/blog.js]: Yea, DJ would have more info on that

**Pull request closed without merging**